demo-shell/src/app.config.json

```diff
@@ -322,6 +322,23 @@
           }
         }
       },
+      {
+        "id": "createdModifiedDateRange",
+        "name": "Date",
+        "enabled": true,
+        "component": {
+          "selector": "date-range-advanced",
+          "settings": {
+            "dateFormat": "dd-MMM-yy",
+            "maxDate": "today",
+            "field": "cm:created, cm:modified",
+            "displayedLabelsByField": {
+              "cm:created": "Created Date",
```

Comment:
Do you pass the label itself as part of the configuration? We shouldn't do it as they won't be localized in this case

Reply:
We've passed in labels here directly as this is the app.config.json file for demo-shell. The component, however, does support passing in i18n ids for localisation. See [here](https://github.com/Alfresco/alfresco-ng2-components/pull/8714/files#diff-b152f75be5c921d39b9ab797301a7146d0e42daa073349892ba276b4c33a55e0R35)